test/puppetlabs/puppetdb/command_test.clj

```diff
@@ -172,45 +172,44 @@
 (defn make-cmd
   "Create a command pre-loaded with `n` attempts"
   [n]
-  {:command nil :version nil :annotations {:attempts (repeat n {})}})
+  {:headers {:id "foo-id-1"
+             :received (tfmt/unparse (tfmt/formatters :date-time) (now))}
+   :body (json/generate-string  {:command "some catalog" :payload "foo" :version 10 :annotations {:attempts (repeat n {})}})})
```

Comment:
Hah - yeah, I was going to suggest that even if you're not going to be me, there might still be a few lines worth shortening a bit.